src/plugins/workspace/public/components/workspace_creator/workspace_creator.tsx

```diff
@@ -53,16 +54,28 @@ export const WorkspaceCreator = (props: WorkspaceCreatorProps) => {
     onlyAllowEssentialEnabled: true,
   });
 
-  const defaultSelectedUseCase = availableUseCases?.[0];
-  const defaultWorkspaceFormValues: Partial<WorkspaceFormSubmitData> = {
-    color: euiPaletteColorBlind()[0],
-    ...(defaultSelectedUseCase
-      ? {
-          name: defaultSelectedUseCase.title,
-          features: [getUseCaseFeatureConfig(defaultSelectedUseCase.id)],
-        }
-      : {}),
-  };
+  const location = useLocation();
+
+  const defaultWorkspaceFormValues = useMemo(() => {
+    let defaultSelectedUseCase;
+    const params = new URLSearchParams(location.search);
+    const useCaseTitle = params.get('useCase');
```

Comment:
Shall we use id instead of title in workspace creator page?

Reply:
No, we should use title to display and search, because query would populate the search key in the input, if we use id, the search query would be `all` which is inconsistent with the `Analytics (all)` in the column.

Reply:
This is workspace creator page right? Not workspace list page. We are `select` an option instead of `search` by title.